clusters/app.ci/prow/02_config/_config.yaml

```diff
@@ -104,12 +104,18 @@ branch-protection:
             users: null
         knative-client:
           branches:
+            pr/olm-fix-release-next:
+              protect: false
+            pr/release-next-fix:
+              protect: false
+            pr/release-next-olm-fix:
+              protect: false
```

Comment:
trying to delete these 3 branches, though I  still get 'protected branch hook declined' error
```
➜  knative-client git:(release-next-ci) ✗ git push openshift :pr/olm-fix-release-next
remote: error: GH006: Protected branch update failed for refs/heads/pr/olm-fix-release-next.
remote: error: Cannot delete this protected branch
To github.com:openshift/knative-client.git
 ! [remote rejected] pr/olm-fix-release-next (protected branch hook declined)
error: failed to push some refs to 'git@github.com:openshift/knative-client.git'
➜  knative-client git:(release-next-ci) ✗ git push openshift :pr/release-next-fix
remote: error: GH006: Protected branch update failed for refs/heads/pr/release-next-fix.
remote: error: Cannot delete this protected branch
To github.com:openshift/knative-client.git
 ! [remote rejected] pr/release-next-fix (protected branch hook declined)
error: failed to push some refs to 'git@github.com:openshift/knative-client.git'
➜  knative-client git:(release-next-ci) ✗ git push openshift :pr/release-next-olm-fix
remote: error: GH006: Protected branch update failed for refs/heads/pr/release-next-olm-fix.
remote: error: Cannot delete this protected branch
To github.com:openshift/knative-client.git
 ! [remote rejected] pr/release-next-olm-fix (protected branch hook declined)
error: failed to push some refs to 'git@github.com:openshift/knative-client.git'
```